### DISCUSSION TITLE
Forecast Grid Data Non-Uniform Valid Time Periods

### DISCUSSION THREAD

**Discussion by aloghides** [weather-gov/api#289] [Q&A]
If for example, you get forecastGridData values for temperature or probability of precipitation in the forecastGridData endpoint you'll notice that the validTime skips random hours instead of giving hourly data.  Is this normal?

Example from my forecast (formatted a bit to make it easier to see the hours):

"validTime": "2020-09-13T   06:00:00+00:00/PT3H",
"validTime": "2020-09-13T   09:00:00+00:00/PT3H",
"validTime": "2020-09-13T   12:00:00+00:00/PT1H",
"validTime": "2020-09-13T   13:00:00+00:00/PT1H",
"validTime": "2020-09-13T   14:00:00+00:00/PT1H",
"validTime": "2020-09-13T   15:00:00+00:00/PT1H",
"validTime": "2020-09-13T   16:00:00+00:00/PT1H",
"validTime": "2020-09-13T   17:00:00+00:00/PT1H",
"validTime": "2020-09-13T   18:00:00+00:00/PT1H",
"validTime": "2020-09-13T   19:00:00+00:00/PT1H",
"validTime": "2020-09-13T   20:00:00+00:00/PT3H",
"validTime": "2020-09-13T   23:00:00+00:00/PT1H",
"validTime": "2020-09-14T   00:00:00+00:00/PT1H",
"validTime": "2020-09-14T   01:00:00+00:00/PT1H",
"validTime": "2020-09-14T   02:00:00+00:00/PT1H",
"validTime": "2020-09-14T   03:00:00+00:00/PT1H",
"validTime": "2020-09-14T   04:00:00+00:00/PT1H",
"validTime": "2020-09-14T   05:00:00+00:00/PT1H",
"validTime": "2020-09-14T   06:00:00+00:00/PT3H",
"validTime": "2020-09-14T   09:00:00+00:00/PT3H",
"validTime": "2020-09-14T   12:00:00+00:00/PT1H",
"validTime": "2020-09-14T   13:00:00+00:00/PT1H",
"validTime": "2020-09-14T   14:00:00+00:00/PT1H",
"validTime": "2020-09-14T   15:00:00+00:00/PT1H",
"validTime": "2020-09-14T   16:00:00+00:00/PT1H",
"validTime": "2020-09-14T   17:00:00+00:00/PT1H",
"validTime": "2020-09-14T   18:00:00+00:00/PT2H",
"validTime": "2020-09-14T   20:00:00+00:00/PT1H",
"validTime": "2020-09-14T   21:00:00+00:00/PT1H",
"validTime": "2020-09-14T   22:00:00+00:00/PT1H",
"validTime": "2020-09-14T   23:00:00+00:00/PT1H"

**Comment by jfbauer432** [comment]
These are time periods.  The "/PT3H" basically means for the next 3 hours.

**Comment by aloghides** [comment]
Thank you!  That makes a lot more sense.   I will close this now.
